include/NovelRT/Maths/QuadTree.h

```diff
@@ -124,8 +127,9 @@ namespace NovelRT::Maths {
              getBottomRight()->tryRemove(point);
     }
 
-    void getIntersectingPoints(const GeoBounds& bounds, std::vector<std::shared_ptr<QuadTreePoint>>& intersectingPoints) {
-      if (getBounds().intersectsWith(bounds)) {
+    //TODO: Why are we returning via a parameter and not the signature's return type?
```

Comment:
Looks like I added a TODO here that was forgotten about. What  should we do?

Reply:
Change it? lul

Reply:
Actually, after reviewing the code this is a correct implementation because the _actual_ `getIntersectingPoints` call returns the correct return type, and this is almost a helper method for when you already have a std::vector already.

Based on this, I'm going to leave as is since I did not originally implement and changing it may break QuadTree.